yw_website/yw_website/apps/website/serializers.py

```diff
@@ -61,3 +66,102 @@ class TagFileSerializer(serializers.ModelSerializer):
     class Meta:
         model = TagFile
         fields = '__all__'
+
+class YesWorkflowSaveSerializer(serializers.ModelSerializer):
+    model = serializers.CharField(required=False, allow_blank=False)
+    model_checksum = serializers.CharField(required=False, allow_blank=True, max_length=128)
+    graph = serializers.CharField(required=False, allow_blank=False)
+    recon = serializers.CharField(required=False, allow_blank=True)
```

Comment:
we might want to make the yes workflow output required

Reply:
Do you want all output to be required?

Reply:
I applied your feedback but this because it is unclear what fields you want to be required.

Reply:
I think the fields that should be required would be
* model
* model_checksum
* graph
* recon
* scripts (ensure there is at least one object)

Do you have any thoughts on this?

Reply:
That's fine with me

Reply:
Done